cmd/vic-machine/configure/configure.go

```diff
@@ -104,7 +105,7 @@ func (c *Configure) Flags() []cli.Flag {
 
 	// flag arrays are declared, now combined
 	var flags []cli.Flag
-	for _, f := range [][]cli.Flag{target, ops, id, compute, container, volume, dns, cNetwork, memory, cpu, certificates, registries, proxies, util, debug} {
+	for _, f := range [][]cli.Flag{target, ops, id, compute, affinity, container, volume, dns, cNetwork, memory, cpu, certificates, registries, proxies, util, debug} {
```

Comment:
could you define this array outside of this for loop?

Reply:
I could, but this is a pattern used consistently across our `vic-machine` commands. I wouldn't object to changing it, but I don't think this PR is the right place to do so.